### ISSUE TITLE
Hourly conversion rate is not working 

### ISSUE THREAD

**Issue by aminlatifi** [Giveth/feathers-giveth#301]
Develop cannot set `usdValue` of new donations. Maybe new update on hourly conversion rate be the cause.
giveth/giveth-dapp#1566
#287 
#298

**Comment by aminlatifi** [comment]
Feathers log on develop:
```
6|feathers | 2020-12-12 03:35: error: Error setting usdValue for donation: 5fd43a9277a32e33c7fce3baFailed to retrieve cryptocompare rate for ts: 1607742000 Error: Failed to retrieve cryptocompare rate for ts: 1607742000
6|feathers |     at getHourlyRateCryptocompare (/home/deploy/feathers-giveth-develop/src/services/conversionRates/getConversionRatesService.js:217:22)
6|feathers |     at <anonymous> 
```

**Comment by aminlatifi** [comment]
The `usdValue` for this should be 2
![image](https://user-images.githubusercontent.com/5684607/101971840-882f1c00-3c49-11eb-8280-ac61f8e7f8f3.png)

**Comment by aminlatifi** [comment]
The `usdValue` for this should be 2
![image](https://user-images.githubusercontent.com/5684607/101971859-a3019080-3c49-11eb-8da9-b382e1498c5a.png)

**Comment by mdehghani** [comment]
this bug is fixed in the pull request #300

**Comment by mohammadranjbarz** [comment]
@mahdine 

I think it should not fix this issue,  in below code
https://github.com/Giveth/feathers-giveth/blob/09275709e72c1d093087091df065fc3ef84a37bd/src/services/donations/donations.hooks.js#L137

we should pass `token.rateEqSymbol || token.symbol` not `token.symbol`, otherwise we should face error with tokens like `XDAI`
I'm working on this issue, and you are welcome to help me ( but I think I found the problem , if I'm not mistaking)
